index.bs

```diff
@@ -165,6 +165,42 @@ the user-agent should retain the previously computed layout state if possible.
   [=skipped=] is a rendering performance optimization.
 </div>
 
+Accessibility {#accessibility}
+=============
+
+Similar to the way 'subtree-visibility' affects painted output of the element's
+subtree, it also affects information exposed to the
+<a href="https://w3c.github.io/css-aam/#dfn-accessibility-tree">accessibility
+tree</a> and <a href="https://w3c.github.io/css-aam/#dfn-assistive-technology">
+assistive technologies</a>:
+* The structure of the page should be exposed to assistive technologies.
```

Comment:
Perhaps flesh this out a little more, something like

* Subtrees with a `subtree-visibility` value of `auto` or `visible` should be included in the accessibility tree as usual.
* Subtrees with a `subtree-visibility` value of `skipped` should be excluded from the accessibility tree.

Reply:
Done.